configure

```diff
@@ -1,12 +1,11 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.71 for streamly 0.8.0.
+# Generated by GNU Autoconf 2.69 for streamly 0.8.1.
```

Comment:
Why use an older autconf version? I guess if we use the same version, we will not have many changes in this file.

Reply:
Ah yes.